src/ansys/pyensight/core/utils/omniverse_cli.py

```diff
@@ -276,7 +282,7 @@ def run_monitor(self):
 
         # Build the Omniverse connection
         omni_link = ov_dsg_server.OmniverseWrapper(
-            destination=self._destination, line_width=self._line_width, use_lines=self._use_lines
+            destination=self._destination, line_width=self.line_width
```

Comment:
Does self.line_width work because it is a getter?  
Or is this a mistake, and you need to use self.line_width() or self._line_width?